presto-druid/src/main/java/com/facebook/presto/druid/DruidClient.java

```diff
@@ -74,6 +91,8 @@
     private final URI druidCoordinator;
     private final URI druidBroker;
     private final String druidSchema;
+    protected final boolean caseInsensitiveNameMatching;
+    private final Cache<SchemaTableName, Optional<RemoteTableObject>> remoteTables;
```

Comment:
I assume the cache you're using is thread-safe, correct me if I'm wrong.

Reply:
@beinan Yes, that's right the cache is thread-safe.

Reply:
Cool, thanks for the clarification.